lint

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+while true; do
```

Comment:
useless linting command for conintous iterations

Reply:
just to be clear this is a separate script that developer can manually call, and have a linting running whenever a file changes.  This isn't tied into ant etc.